src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -424,7 +424,7 @@ public static FormulaValue Split(IRContext irContext, StringValue[] args)
             var substrings = string.IsNullOrEmpty(separator) ? text.Select(c => new string(c, 1)) : text.Split(new string[] { separator }, StringSplitOptions.None);
             var rows = substrings.Select(s => new StringValue(IRContext.NotInSource(FormulaType.String), s));
 
-            return new InMemoryTableValue(irContext, StandardSingleColumnTableFromValues(irContext, rows.ToArray(), forceSingleColumn: true, columnName: BuiltinFunction.OneColumnTableResultName));
+            return new InMemoryTableValue(irContext, StandardSingleColumnTableFromValues(irContext, rows.ToArray(), forceSingleColumn: true, columnName: ((TableType)irContext.ResultType).SingleColumnFieldName));
```

Comment:
>((TableType)irContext.ResultType).SingleColumnFieldName) [](http://example.com/codeflow?start=153&length=56)

Will add helper method

Reply:
note we're already passing in irContext  to this function... could we completely remove columnName param?

Reply:
Let's remove that param, good idea